cmd/root/root.go

```diff
@@ -45,6 +45,16 @@ func NewRootCommand() (*cobra.Command, error) {
 	cmd.AddCommand(insights.NewInsightsCommand())
 	cmd.AddCommand(version.NewVersionCommand())
 
+	errors := cmd.Flags().MarkHidden(constants.FlagNameEndpoint)
+	if errors != nil {
+		return nil, errors
+	}
+
+	errors = cmd.Flags().MarkHidden(constants.FlagNameBeta)
+	if errors != nil {
+		return nil, errors
+	}
+
```

Comment:
Usually the convention is to call the error value that gets returned `err`. Calling it `errors` sort of makes it sound like there are multiple errors being returned when in reality it'd only ever be one error 

```suggestion
	err := cmd.Flags().MarkHidden(constants.FlagNameEndpoint)
	if err != nil {
		return nil, err
	}

	err = cmd.Flags().MarkHidden(constants.FlagNameBeta)
	if err != nil {
		return nil, err
	}

```

Reply:
Just a note: there's also a pretty low likelihood that `cmd.Flags().MarkHidden` would even return an error. So this is the right way to handle it: just bubble it up.

I think the main case these would return an error is if `constants.FlagName...` wasn't defined in the array of command flags. And in that case, it'd be more a programmer error vs. a user error or system failure and you'd see it right away